tests/logs/set_maintenance/test_set_maintenance/test_maintenance_off.txt

```diff
@@ -0,0 +1,9 @@
+[    INFO] Get maintenance report: started
+[    INFO] Get maintenance report: finished
+[    INFO] Adjust maintenance report: started
+[    INFO] Following repositories will be removed from maintenance mode: 
+repo2
```

Comment:
I think this is showing that it would be quite nice to format this somehow, e.g. indent it.

On the real production system, the prefix of each log message is going to be even deeper (with a timestamp as well), so this is going to look very disconnected, arguably quite ugly as the "Following repositories..." message is deeply indented but then each repo is not.

What do you think?  Maybe play around with either indenting this by a fair number of spaces, or else make each repo ID have its own call to the logger?

Reply:
yeah, I agree. 
I actually tried to use ' - ' as indent, but it didn't look better. I also tried to make each repo ID have its own call to the logger, but still didn't improve much... I guess I would need at least one more commit to finish this PR, so put it on hold then. Maybe combine them is the answer.